.github/workflows/CI.yml

```diff
@@ -53,10 +53,10 @@ jobs:
         - { os: ubuntu-22.04  , ocaml-version: 4.14.x }
         - { os: ubuntu-22.04  , ocaml-version: "ocaml-variants.4.14.2+options,ocaml-option-32bit", publish: true, fnsuffix: -ubuntu-i386 }
         - { os: ubuntu-20.04  , ocaml-version: 4.14.x }
-        - { os: windows-2022  , ocaml-version: "ocaml.4.14.2,system-mingw"              , publish: true  , fnsuffix: -windows-x86_64 }
-        - { os: windows-2019  , ocaml-version: "ocaml.4.14.2,system-mingw,arch-x86_32"  , publish: true  , fnsuffix: -windows-i386 }
-        - { os: windows-2022  , ocaml-version: "ocaml.4.14.2,system-msvc" }
-        - { os: windows-2019  , ocaml-version: "ocaml.4.14.2,system-msvc,arch-x86_32" }
+        - { os: windows-2022  , ocaml-version: "ocaml.4.14.2,system-mingw" }
+        - { os: windows-2019  , ocaml-version: "ocaml.4.14.2,system-mingw,arch-x86_32" }
+        - { os: windows-2022  , ocaml-version: "ocaml.4.14.2,system-msvc"               , publish: true  , fnsuffix: -windows-x86_64 }
+        - { os: windows-2019  , ocaml-version: "ocaml.4.14.2,system-msvc,arch-x86_32"   , publish: true  , fnsuffix: -windows-i386 }
```

Comment:
mingw is a kind of emulation implementation and apparently stuck with the old MS CRT (https://www.msys2.org/docs/environments/#msvcrt-vs-ucrt). While we haven't seen any problems arising from that directly (that I know of), I'm not sure we should prefer it to the full native build if one's available. But then again, this is pretty much the extent of my knowledge about mingw and MSVC, so maybe all this is irrelevant.

Reply:
Since there are no strong technical arguments, if you feel strongly about this then we can remove these lines from the patch. I have a feeling that long term it would still be better to prefer a full-native toolchain, but I may be completely mistaken about this. As said, I am no expert when it comes to either mingw or MSVC.

Reply:
how about we just publish both?   It doesn't seem to hurt, and then people can try both if having issues.

Reply:
There's a risk of getting many questions like "which one do I use?"